lib/active_shipping/carriers/fedex.rb

```diff
@@ -557,22 +562,23 @@ def extract_address(document, possible_node_names)
     end
 
     def extract_timestamp(document, node_name)
-      if timestamp_node = document.elements[node_name]
-        Time.parse(timestamp_node.to_s).utc
+      if timestamp_node = document.at(node_name)
+        if timestamp_node.text =~ /\A(\d{4}-\d{2}-\d{2})T00:00:00\Z/
+          Date.parse($1)
+        else
+          Time.parse(timestamp_node.text)
+        end
```

Comment:
From API manual: "Time package was shipped/tendered over to FedEx. Time  portion will be populated if available, otherwise will be set to midnight"